pkg/awsclient/mock/mock_client.go

```diff
@@ -1,879 +1,39 @@
-// Code generated by MockGen. DO NOT EDIT.
-// Source: pkg/awsclient/client.go
-
-// Package mock is a generated GoMock package.
 package mock
```

Comment:
Note to reviewers: This is a new file. It just happens to have the same name as the generated file used to. Ignore the "delta".